requirements.txt

```diff
@@ -0,0 +1,42 @@
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile
+#
+-e file:./processor
+-e file:./prio
+attrs==19.3.0             # via jsonschema, pytest
+cachetools==4.0.0         # via google-auth
+certifi==2019.11.28       # via requests
+chardet==3.0.4            # via requests
+click==7.1.1
+decorator==4.4.2          # via gcsfs
+gcsfs==0.2.3
+google-auth-oauthlib==0.4.1  # via gcsfs
+google-auth==1.11.2       # via gcsfs, google-auth-oauthlib
+idna==2.9                 # via requests
+importlib-metadata==1.5.0  # via jsonschema, pluggy, pytest
+jsonschema==3.2.0
+more-itertools==8.2.0     # via pytest
+oauthlib==3.1.0           # via requests-oauthlib
+packaging==20.3           # via pytest
+pluggy==0.13.1            # via pytest
+py4j==0.10.7              # via pyspark
+py==1.8.1                 # via pytest
+pyasn1-modules==0.2.8     # via google-auth
+pyasn1==0.4.8             # via pyasn1-modules, rsa
+pyparsing==2.4.6          # via packaging
+pyrsistent==0.15.7        # via jsonschema
+pyspark==2.4.5
+pytest==5.3.5
+requests-oauthlib==1.3.0  # via google-auth-oauthlib
+requests==2.23.0          # via gcsfs, requests-oauthlib
+rsa==4.0                  # via google-auth
+six==1.14.0               # via google-auth, jsonschema, packaging, pyrsistent
+urllib3==1.25.8           # via requests
+wcwidth==0.1.8            # via pytest
+zipp==3.1.0               # via importlib-metadata
+
+# The following packages are considered to be unsafe in a requirements file:
+# setuptools
```

Comment:
This warning looks weird and unexpected...

Reply:
This also looks like it's relevant to the error message I've seen: `pkg_resources.ContextualVersionConflict: (setuptools 39.2.0 (/usr/lib/python3.6/site-packages), Requirement.parse('setuptools>=40.3.0'), {'google-auth'})`